homeassistant/components/camera/verisure.py

```diff
@@ -93,8 +89,12 @@ def delete_image(self, event):
                                     '{}{}'.format(
                                         self._image_id,
                                         '.jpg'))
-        _LOGGER.debug('Deleting old image %s', remove_image)
-        os.remove(remove_image)
+        try:
+            os.remove(remove_image)
+            _LOGGER.debug('Deleting old image %s', remove_image)
+        except OSError as error:
+            if error.errno != 2:
```

Comment:
Use errno.ENOENT instead. You might have to import errno at the top of the file. Using "magic numbers" makes the code harder to read and maintain.